src/Microsoft.DotNet.Darc/Darc/Operations/VirtualMonoRepo/ResolveConflictOperation.cs

```diff
@@ -0,0 +1,215 @@
+        IProductConstructionServiceApi pcsApiClient,
+        ILogger<ResolveConflictOperation> logger)
+    : CodeFlowOperation(options, forwardFlower, backFlower, backflowConflictResolver, vmrInfo, vmrCloneManager, dependencyTracker, dependencyFileManager, localGitRepoFactory, barApiClient, fileSystem, logger)
+{
+    private readonly ResolveConflictCommandLineOptions _options = options;
+    private readonly IVmrInfo _vmrInfo = vmrInfo;
+    private readonly IProcessManager _processManager = processManager;
+    private readonly ILocalGitRepoFactory _localGitRepoFactory = localGitRepoFactory;
+    private readonly IBasicBarClient _barClient = barApiClient;
+    private readonly IProductConstructionServiceApi _pcsApiClient = pcsApiClient;
+    private readonly ILogger<ResolveConflictOperation> _logger = logger;
+
+    protected override async Task ExecuteInternalAsync(
+        string repoName,
+        string? sourceDirectory,
+        IReadOnlyCollection<AdditionalRemote> additionalRemotes,
+        CancellationToken cancellationToken)
+    {
+        if (string.IsNullOrEmpty(_options.SourceRepo))
+        {
+            throw new ArgumentException("Please specify a local path on disk to the source git repo" +
+                " or VMR to flow from.");
+        }
+
+        var subscription = await FetchCodeflowSubscriptionAsync(_options.SubscriptionId);
+
+        var pr = await _pcsApiClient.PullRequest.GetTrackedPullRequestBySubscriptionIdAsync(
+            subscription.Id.ToString(),
+            cancellationToken);
+
+        ValidateConflictingPrAsync(pr);
+
+        var buildId = GetBuildIdFromTrackedPr(pr, subscription.Id);
+
+        var sourceGitRepoPath = new NativePath(_processManager.FindGitRoot(_options.SourceRepo));
+        var targetGitRepoPath = new NativePath(_processManager.FindGitRoot(Directory.GetCurrentDirectory()));
+
+        await ValidateLocalVmr(subscription);
+        await ValidateLocalRepo(subscription);
+
+        await ValidateLocalBranchMatchesRemote(targetGitRepoPath, pr.HeadBranch);
+
+        _vmrInfo.VmrPath = string.IsNullOrEmpty(subscription.TargetDirectory)
+            ? targetGitRepoPath
+            : sourceGitRepoPath;
+
+        Console.WriteLine("Ready to execute codeflow locally. Proceed? y/n");
+
+        var confirmation = Console.ReadLine();
+
+        if (confirmation != "y")
+        {
+            Console.WriteLine("Aborting resolve operation...");
+            return;
+        }
+        else
+        {
+            Console.WriteLine("Proceeding with codeflow...");
+        }
+
+        await FlowCodeLocallyAsync(
+            targetGitRepoPath,
+            isForwardFlow: !string.IsNullOrEmpty(subscription.TargetDirectory),
+            additionalRemotes,
+            cancellationToken,
+            buildId: buildId);
+
+        Console.WriteLine("Codeflow has finished. Please resolve the conflicts and commit the changes. " +
+            "Then run git vmr resolve --continue.");
```

Comment:
```suggestion
            "Then run `darc vmr resolve --continue`");
```
should probably be this?